testdata/go/v4-alpha/memcached-operator/test/e2e/e2e_test.go

```diff
@@ -76,6 +86,26 @@ var _ = Describe("memcached", func() {
 				"pod-security.kubernetes.io/enforce=restricted")
 			_, err = utils.Run(cmd)
 			Expect(err).To(Not(HaveOccurred()))
+
+			By("uncommenting all sections with 'monitoring' to enable operator custom metrics")
+			cmd = exec.Command("sed", "-i",
+				`s/\/\/"github.com\/example\/memcached-operator\/monitoring"/"github.com\/example\/memcached-operator\/monitoring"/g`,
+				"controllers/memcached_controller.go", "main.go")
+			_, err = utils.Run(cmd)
+			Expect(err).To(Not(HaveOccurred()))
+
+			cmd = exec.Command("sed", "-i",
```

Comment:
We are not calling goling to test the code but we should not have this big lines.
It is not a good practice need to scroll to check the code. Could you please break the lines for maintainability and readbality?

Reply:
Assigned these strings to constants, but there are still long lines, as these strings are given as a single argument for the command. https://github.com/operator-framework/operator-sdk/blob/61dcef47b5f924ca240d0fed585927ccae245e7e/testdata/go/v4-alpha/memcached-operator/test/e2e/e2e_test.go#L46 Please let me know if it's still an issue.

Reply:
Yes, that is an issue for mainatainability and supportability.
Note that sed does not work in all OS for example

What we should to do here is copy and paste https://github.com/kubernetes-sigs/kubebuilder/blob/b5899413b689914cb672cb2dd07deafcd8a01176/pkg/plugin/util/util.go#L194-L215 the replace implementation to scaffold it in the utils file on this test.

Therefore, it also explains how to do it in a much more readable way.

Reply:
Thanks, Done